rpa-home/modules/ROOT/pages/automation-tutorial-composer.adoc

```diff
@@ -0,0 +1,67 @@
+
+== Step 8.1: Create a New Composer Flow
+
+To create a new Composer flow:
+
+. Open MuleSoft Composer.
++
+If you don't know what's your organization's MuleSoft Composer instance URL, contact your organization administrator for details. 
+. In the menu bar, click *Composer Flows*.
+. Click *New Flow*.
+. Click the edit button next to *Untitled Flow* and provide a name for this flow, for example, *Trigger RPA Automation*.
+
+== Step 8.2: Add a Start Event for the Flow
+
+Every Composer flow needs a start event to trigger the execution. To add a starting action: 
+
+. In the *What Should Start This Flow?* window, click *Scheduler*. 
++
+The *Scheduler* triggers the execution of the flow every certain amount of minutes that you can configure. 
++
+image:composer-trigger.png[The action card showing options to configure the flow trigger in Composer, 50%, 50%]
+. Leave the *Every* field in its default value: *15 minutes*.
+
+== Step 8.3: Connect Composer with MuleSoft RPA
+
+Before you can call an RPA process from Composer, you must configure the connection between the two systems. To configure the connection between Composer and MuleSoft RPA: 
+
+. Click *Add* (image:add-icon.png[The Add Event button, 2%, 2%]), next to the Scheduler event you previously created. 
+. Select *System Action*. 
+. Select *MuleSoft RPA*. 
++ 
+Because this is the first time you interact with a new system (MuleSoft RPA), Composer opens the connection panel.
+. Provide the following details to connect with your MuleSoft RPA account: 
+** *Connection Display Name*: A name for the connection, for example, *My RPA Account*. 
+** *RPA API URL*: The RPA API URL of your RPA Manager instance. This URL is formed by appending `/rpa/api/v1` to the URL of RPA Manager, for example: `https://<tenant-name>.rpa.mulesoft.com/rpa/api/v1`.
+** *User API Key*: An API key to connect MuleSoft RPA with Composer. Use the same API Key you created as part of the tutorial prerequisites. You can create a new user API key by following the steps in xref:rpa-manager::usermanagement-connect.adoc#create-an-api-key-for-a-user[Create an API Key for a User].
+. Click *Connect*. 
+
+Composer shows a message confirming the connection was successful. If you see an error message while creating the connection, verify the configuration details and try again. 
+
+== Step 8.4: Add an Action to Invoke the RPA Process
+
+After successfully configuring the connection to MuleSoft RPA, follow these steps to invoke an RPA process:
+
+. In the MuleSoft RPA action card, select *Invoke RPA Process* from the *Action* dropdown menu. 
+. In the *RPA Process* dropdown menu, select the invokable run configuration you created in Step 7.1. 
```

Comment:
Make the step a link?

Reply:
Do you need to add the "dropdown menu" bits in these two?
Select *Action* > *Invoke RPA Process*...
From *RPA Process*, select...